### DISCUSSION TITLE
Use outline to adjust graph

### DISCUSSION THREAD

**Discussion by hahnpv** [slnsys/obsidian-canvas2document#7] [Ideas]
If we assume the user has a well-structured canvas (perhaps using a plugin like "canvas mindmap") it would be interesting if canvas2document could use changes to the outline on the longform document to alter the structure of the canvas accordingly.

**Comment by slnsys** [comment]
Canvas2Document is designed as a one-way path from canvas to document, especially for the process of creating an article from visual brainstorming to a long-form document.

There are simply too many different ways in which the structure could have been in the document before.

This feature sounds like the use case for a separate plugin that creates a dynamic, permanent synchronization of MD-Doc and Canvas.
